lightningd/plugin_control.c

```diff
@@ -0,0 +1,132 @@
+#include <ccan/tal/path/path.h>
+#include <common/json_command.h>
+#include <common/jsonrpc_errors.h>
+#include <common/param.h>
+#include <dirent.h>
+#include <errno.h>
+#include <lightningd/plugin_control.h>
+#include <lightningd/plugin_hook.h>
+#include <unistd.h>
+
+/**
+ * A plugin command which permits to control plugins without restarting
+ * lightningd. It takes a subcommand, and an optional subcommand parameter.
+ */
+static struct command_result *json_plugin_control(struct command *cmd,
+						const char *buffer,
+						const jsmntok_t *obj UNNEEDED,
+						const jsmntok_t *params)
+{
+	const char *subcmd;
+	subcmd = param_subcommand(cmd, buffer, params,
+							  "start", "stop", "startdir", "reload", "list", NULL);
```

Comment:
Weird indent here: do you have tabs set to 4 spaces?

Reply:
No, but when aligning I'm tricked by tabs